src/icons/api-docs.svg

```diff
@@ -0,0 +1,13 @@
+<svg viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M5 15.7786V3.58789H16L19 7.58789V21.5879H16.5408" stroke="white" stroke-linecap="round" stroke-linejoin="round"/>
+<g opacity="0.3">
+<mask id="path-2-inside-1_716_7726" fill="white">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9 9.58789C8.72386 9.58789 8.5 9.81175 8.5 10.0879C8.5 10.364 8.72386 10.5879 9 10.5879H15C15.2761 10.5879 15.5 10.364 15.5 10.0879C15.5 9.81175 15.2761 9.58789 15 9.58789H9ZM9 12.5879C8.72386 12.5879 8.5 12.8117 8.5 13.0879C8.5 13.364 8.72386 13.5879 9 13.5879H15C15.2761 13.5879 15.5 13.364 15.5 13.0879C15.5 12.8117 15.2761 12.5879 15 12.5879H9Z"/>
+</mask>
+<path d="M9.5 10.0879C9.5 10.364 9.27614 10.5879 9 10.5879V8.58789C8.17157 8.58789 7.5 9.25946 7.5 10.0879H9.5ZM9 9.58789C9.27614 9.58789 9.5 9.81175 9.5 10.0879H7.5C7.5 10.9163 8.17157 11.5879 9 11.5879V9.58789ZM15 9.58789H9V11.5879H15V9.58789ZM14.5 10.0879C14.5 9.81175 14.7239 9.58789 15 9.58789V11.5879C15.8284 11.5879 16.5 10.9163 16.5 10.0879H14.5ZM15 10.5879C14.7239 10.5879 14.5 10.364 14.5 10.0879H16.5C16.5 9.25946 15.8284 8.58789 15 8.58789V10.5879ZM9 10.5879H15V8.58789H9V10.5879ZM9.5 13.0879C9.5 13.364 9.27614 13.5879 9 13.5879V11.5879C8.17157 11.5879 7.5 12.2595 7.5 13.0879H9.5ZM9 12.5879C9.27614 12.5879 9.5 12.8117 9.5 13.0879H7.5C7.5 13.9163 8.17157 14.5879 9 14.5879V12.5879ZM15 12.5879H9V14.5879H15V12.5879ZM14.5 13.0879C14.5 12.8117 14.7239 12.5879 15 12.5879V14.5879C15.8284 14.5879 16.5 13.9163 16.5 13.0879H14.5ZM15 13.5879C14.7239 13.5879 14.5 13.364 14.5 13.0879H16.5C16.5 12.2595 15.8284 11.5879 15 11.5879V13.5879ZM9 13.5879H15V11.5879H9V13.5879Z" fill="white" mask="url(#path-2-inside-1_716_7726)"/>
+</g>
+<rect x="3.30312" y="16.0121" width="12.9522" height="6.62146" rx="0.65" stroke="white" stroke-width="0.7"/>
+<path d="M6.59144 21.0112H5.7793L6.83417 17.8848H7.84019L8.89505 21.0112H8.08291L7.34863 18.6725H7.3242L6.59144 21.0112ZM6.48305 19.7808H8.18061V20.3548H6.48305V19.7808Z" fill="white"/>
+<path d="M9.4638 21.0112V17.8848H10.7553C10.9894 17.8848 11.1914 17.9306 11.3613 18.0222C11.5323 18.1127 11.6641 18.2394 11.7567 18.4023C11.8493 18.5641 11.8957 18.7524 11.8957 18.9671C11.8957 19.1829 11.8483 19.3717 11.7537 19.5335C11.66 19.6943 11.5262 19.8189 11.3522 19.9075C11.1782 19.996 10.9716 20.0403 10.7324 20.0403H9.93552V19.4449H10.5919C10.7059 19.4449 10.8011 19.4251 10.8774 19.3854C10.9548 19.3457 11.0133 19.2902 11.053 19.219C11.0927 19.1467 11.1125 19.0628 11.1125 18.9671C11.1125 18.8704 11.0927 18.787 11.053 18.7168C11.0133 18.6455 10.9548 18.5906 10.8774 18.5519C10.8001 18.5132 10.7049 18.4939 10.5919 18.4939H10.2195V21.0112H9.4638Z" fill="white"/>
+<path d="M13.2559 17.8848V21.0112H12.5003V17.8848H13.2559Z" fill="white"/>
```

Comment:
In case we need the icons to. be a different color, can use currentColor. See https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

Reply:
updated

Reply:
I think this is supposed to be for all fills where the path is. Such as line 12. This is what I had done for run-step.svg

Reply:
I see your point, a question here though, some icons have multi `paths` with different colors, do we still change them all into `currentColor`?

Reply:
tried changing all fills to be `currentColor`, it seems off when there are more than 1 color in the svg file, so I'm only modifying one color svg files to be `currentColor`:
example, when I changed the `help-center` to be all `currentColor`:
![Screen Shot 2022-02-07 at 1 28 00 PM](https://user-images.githubusercontent.com/80717697/152858562-cba212cf-17f9-41f3-a51c-205e5ce43f02.png)

Reply:
as well as, `university`, when remove `fill=none`, it'll be off:
![Screen Shot 2022-02-08 at 9 54 17 AM](https://user-images.githubusercontent.com/80717697/153025879-f93f8b9d-509c-4241-8330-ac0ca501904e.png)